version/version_test.go

```diff
@@ -0,0 +1,27 @@
+package version
+
+import (
+	"encoding/json"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+type rpcChainCompatibility struct {
+	RPCChainVMProtocolVersion map[string]uint `json:"rpcChainVMProtocolVersion"`
+}
+
+const compatibilityFile = "../compatibility.json"
```

Comment:
I think we should handle this the same way we did in avalanchego

Reply:
I think it's a better idea for the compatibility file to live at the top level as I've written it. That way it's more language agnostic, especially since we may want to add rust vms in the near future. I'd like to only have one standard approach for defining this. The way I did it is the same way we did it in subnet-evm as well.

Reply:
I think then it probably makes sense to move this test to the root of the repo - embed the file there - and test that.

Reply:
ok, done